### DISCUSSION TITLE
Branches in pages?

### DISCUSSION THREAD

**Discussion by ajsirota** [community/community#22131] [Other Feature Feedback, Questions, & Ideas]
<p>Is there a way to have a Pages site on a branch? I’m using Pages to host a web app, and it doesn’t seem like I can test branches on Pages.</p>
<p>Thank you in advance.</p>

**Comment by ncoleman** [comment, marked as answer]
<p>Pages will only publish from master.</p>

**Reply by orubel** [reply]
Yep and mine has always worked but when I updated it recently, the update is not showing https://github.com/Beapi-io/beapi-docs

**Comment by ajsirota** [comment]
<p>Okay, thank you for letting me know. :slightly_frowning_face:</p>
